android/src/main/java/com/reactlibrary/Helpers.kt

```diff
@@ -98,6 +101,32 @@ internal fun getJudoConfiguration(type: PaymentWidgetType, options: ReadableMap)
             .build()
 }
 
+internal fun getAuthorization(options: ReadableMap): Authorization {
+    val token = options.authorization?.getString("token")
+
+    options.authorization?.hasKey("secret").let {
+        if (it == true) {
+            val secret = options.authorization?.getString("secret")
```

Comment:
since [getString](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/bridge/ReadableMap.java#L32) can return null we can do:
```
val secret = options.authorization?.getString("secret")?.let {
    return BasicAuthorization.Builder()
                    .setApiToken(token)
                    .setApiSecret(secret)
                    .build()
}
```
and remove:
```
options.authorization?.hasKey("secret").let {
        if (it == true) {
                 ...
       }
}
```